src/server/Document/component.test.jsx

```diff
@@ -35,8 +35,9 @@ describe('Document Component', () => {
   );
   const styleTagsAmp = <style amp-custom="">{'html { color: red; }'}</style>;
 
-  const shallowDocument = ({ service, isAmp }) =>
-    shallowRender(
+  const shallowDocument = ({ service, isAmp, testName }) =>
```

Comment:
i don't think this should be called shallowDocument anymore, because we are no longer shsallowRendering